src/list.js

```diff
@@ -0,0 +1,13 @@
+export default class List {
+  constructor() {
+    this.list = [];
```

Comment:
- [ ] The array of  to-do tasks should be an array of objects with each object containing three keys - description [string], completed [bool], index: [number]